src/containers/TemplateBuilder/template/General/Filters.tsx

```diff
@@ -191,6 +216,50 @@ const Filters: React.FC = () => {
           </div>
         </div>
       )}
+      <Header as="h3">Dashboard Restrictions</Header>
+      <p className="smaller-text">
+        Template will only appear on Dashboard if the following filters have active applications (no
+        restrictions means it'll always appear)
+      </p>
+      <div className="flex-row-start-center">
+        <DropdownIO
+          value={selectedRestriction?.id ?? 0}
+          placeholder="Select filter"
+          title="Filters"
+          options={allFilters.filter(
+            (fil) => !template.dashboardRestrictions?.includes(fil?.code ?? '')
+          )}
+          getKey={'id'}
+          getValue={'id'}
+          getText={'title'}
+          setValue={(_, fullValue) => {
+            setSelectedRestriction(fullValue)
+            setSelectedRestrictFilter(undefined)
+          }}
+          additionalStyles={{ marginBottom: 0 }}
+        />
+        {selectedRestriction && (
+          <Icon className="clickable" name="add square" onClick={addDashboardRestriction} />
+        )}
+        {selectedRestrictFilter && (
+          <Icon className="clickable" name="minus square" onClick={removeDashboardRestriction} />
+        )}
+      </div>
+      <div className="spacer-20" />
+      <div className="filter-joins">
+        {template.dashboardRestrictions &&
+          template.dashboardRestrictions.map((filter) => (
+            <Label
+              key={filter}
+              onClick={() => {
+                setSelectedRestrictFilter(filter)
+              }}
```

Comment:
This is not great UI, but it's on a par with (and uses the same elements) as the existing filter edit UI. One of these days I wanna re-write the whole thing!

Reply:
What is bad about it? Is this it this one in particular, or a number of UI components?

Reply:
The template builder overall, in particular all the  "IO" components with the label/contents pattern. That was never part of the original design and while it looks okay in moderation, a whole page of it (especially when nested) looks pretty bleh (IMHO), not to mention it's definitely not mobile friendly. 
![Screenshot 2023-08-04 at 1 55 13 PM](https://github.com/openmsupply/conforma-web-app/assets/5456533/5c16edd9-f8e2-4f7f-84b7-4cfb99da8249)